ckanext/canada/templates/public/home/index.html

```diff
@@ -1,2 +1,2 @@
 {# hide CKAN front page on public site #}
-{{ h.redirect_to(controller='package', action='search') }}
+{{ h.redirect_to('package.search') }}
```

Comment:
same question about these `package.search` urls: do they work? should they be dataset?

Reply:
All `package` stuff has been replaced with `dataset` blueprint or the `pkg.type` blueprint.